OPHD/States/MapViewState.h

```diff
@@ -246,6 +247,7 @@ class MapViewState : public Wrapper
 private:
 	MainReportsUiState& mMainReportsState;
 	TileMap* mTileMap = nullptr;
```

Comment:
Minor note but something to consider going forward, we've been preferring brace initializers versus assignment. This is mostly a stylistic choice (I'm sure there are some differences in the generated code) but going forward brace init is the preferred style.